org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/info/InfoWindow.java

```diff
@@ -873,6 +873,9 @@ protected boolean loadInfoRelatedTabs() {
 				String s_sqlFrom = embedInfo.getFromClause();
 				/** Where Clause						*/
 				String s_sqlWhere = relatedInfo.getLinkColumnName() + "=?";
+				String infoWhere = embedInfo.getWhereClause(); 
+				if(!Util.isEmpty(infoWhere))
+					s_sqlWhere += " AND " + infoWhere;
```

Comment:
safer to enclose infoWhere in `()`, i.e `s_sqlWhere += " AND (" + infoWhere + ")";`